internal/bminventory/inventory_test.go

```diff
@@ -489,14 +489,41 @@ var _ = Describe("GenerateClusterISO", func() {
 				mockS3Client.EXPECT().GetObjectSizeBytes(gomock.Any(), gomock.Any()).Return(int64(100), nil).Times(1)
 				mockEvents.EXPECT().AddEvent(gomock.Any(), *cluster.ID, nil, models.EventSeverityInfo, "Generated image (Image type is \"minimal-iso\", SSH public key is not set)", gomock.Any())
 
-				generateReply := generateClusterISO()
+				generateReply := generateClusterISO(models.ImageTypeMinimalIso)
 				Expect(generateReply).Should(BeAssignableToTypeOf(installer.NewGenerateClusterISOCreated()))
 				_, err := os.Stat(isoFilePath)
 				Expect(os.IsNotExist(err)).To(BeTrue())
 			})
 
 			It("Regenerates the iso for a new type", func() {
-				//TODO
+				Skip("find a solution for waiting 10 seconds")
```

Comment:
Maybe create the image info directly in the cluster initially?

Reply:
Yeah thought about it, but isn't it kinda hackey? Also not sure it'll resolve the issue, as the sleep was added intentionally to avoid an error :/ Maybe I'll give it a try..

Reply:
Looks ok, used this to rollback creation date: https://github.com/openshift/assisted-service/blob/964357bf69967e9b0f7ea80720b78c5ebbb04acd/internal/bminventory/inventory_test.go#L367